### DISCUSSION TITLE
zigbee2mqtt.io dev branch addition not automatically merged into main

### DISCUSSION THREAD

**Discussion by danielyc** [Koenkk/zigbee2mqtt#13699] [General]
Hi @Koenkk,

i've pushed a new [device addition](https://github.com/Koenkk/zigbee2mqtt.io/pull/1512) to the zigbee2mqtt.io dev branch a couple days ago but it wasn't automatically merged correctly into the main branch. Can i make a new pull request for the main branch or can you fix it?

Thanks in advance!

**Comment by Koenkk** [comment]
This will be done with the next release (since the device is only supported in the dev version of z2m yet). Next release is on 1 September.

**Reply by danielyc** [reply]
Alright great!
